src/app/build.gradle

```diff
@@ -24,17 +24,17 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:28.0.0'
-    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.gms:play-services-ads-identifier:16.0.0'
     testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'com.android.support.test:runner:1.0.1'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+    androidTestImplementation 'androidx.test:runner:1.1.0'
```

Comment:
fix to `androidTestImplementation 'androidx.test.ext:junit:1.1.1'`
https://stackoverflow.com/questions/52776716/androidjunit4-class-is-deprecated-how-to-use-androidx-test-ext-junit-runners-an

and fix...

https://github.com/imobile-maio/maio-Android-MoPubMediationAdapter/blob/master/src/app/src/androidTest/java/jp/maio/maioandroidmopubadapter/ExampleInstrumentedTest.java#L17

from
`@RunWith(AndroidJUnit4.class)`
to
`@RunWith(AndroidJUnit4ClassRunner.class)`

https://github.com/imobile-maio/maio-Android-MoPubMediationAdapter/blob/master/src/app/src/androidTest/java/jp/maio/maioandroidmopubadapter/ExampleInstrumentedTest.java#L22

from
`Context appContext = InstrumentationRegistry.getTargetContext();`
to
`Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();`